### PR TITLE
Add new volunteer Satyam Bajpai

### PR DESCRIPTION
**Pull request by satyam73**

### CODE REVIEW

File: data/users/satyam73.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Satyam Bajpai",
+  "bio": "Full Stack Web Developer",
+  "image": "https://avatars.githubusercontent.com/u/85296770",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["javascript", "reactjs", "html", "css", "nodejs", "mongodb"]
```
Comment (RakeshSangem) [inline data/users/satyam73.json:8]: minimize topics 2-3

**Review by RakeshSangem — CHANGES_REQUESTED** [review]

File: data/users/satyam73.json (hunk 1 above)
Comment (satyam73) [reply data/users/satyam73.json]: I have updated topics. Please look into it.

File: data/users/satyam73.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Satyam Bajpai",
+  "bio": "Full Stack Web Developer",
+  "image": "https://avatars.githubusercontent.com/u/85296770",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["frontend", "backend", "javascript"]
```
Comment (codewithvoid) [inline data/users/satyam73.json:8]: @satyam73 would be better to mention frontend and backend technologies that you would like help devs with.

E.g `ReactJS`, `NextJS`, `NodeJS`.

This way devs can ask you more pointed queries.

**Comment by codewithvoid** [comment]
Thanks for volunteering @satyam73 :)

File: data/users/satyam73.json (hunk 2 above)
Comment (satyam73) [reply data/users/satyam73.json]: I have resolved this. Please take a look.

**Comment by satyam73** [comment]
> Thanks for volunteering @satyam73 :)

It's my pleasure to help someone.

**Pull request merged**